policyengine_us/parameters/gov/states/oh/tax/income/credits/exemption/agi_less_exemption_cap.yaml

```
@@ -0,0 +1,10 @@
+description: OH Exemption Credit AGI Limit
+values:
+  2021-01-01: 30_000
+metadata:
+  unit: currency-USD
+  period: year
+  label: OH Exemption Credit
```

And others (spell out Ohio, lowercase elsewhere, ensure it reflects the parameter purpose)
```suggestion
  label: Ohio exemption credit AGI limit
```